### ISSUE TITLE
Text in Markdown Editor should wrap

### ISSUE THREAD

**Issue by AlexandreSajus** [Bycelium/PyFlow#233]
It's kind of weird to be in this situation when editing Markdown in a Markdown block:

![image](https://user-images.githubusercontent.com/62465003/151712962-0aaed119-4de7-4cba-994f-428a3add4842.png)

Where I'm editing a long line and instead of wrapping, the editor takes me to the left, hiding the beginning of the other lines.

Text in Markdown editors should wrap

**Comment by MathisFederico** [comment]
Yeah the whole markdown block should be refactored as mentioned in #153
